nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/status/history/EmbeddedQuestDbRolloverHandlerTest.java

```diff
@@ -198,10 +292,14 @@ private void thenRemainingPartitionsAre(final List<Integer> expectedDays) throws
             existingPartitions.add(new StringBuilder(record.getStr(0)).toString());
         }
 
-        Assert.assertEquals(expectedPartitions.size(), existingPartitions.size());
+        Assert.assertEquals(expectedPartitions.length, existingPartitions.size());
 
         for (final String expectedPartition : expectedPartitions) {
             Assert.assertTrue("Partition " + expectedPartition + " is expected", existingPartitions.contains(expectedPartition));
         }
     }
+
+    private void thenNoPartitionsExpected() throws Exception {
+        thenTheRemainingPartitionsAre();
+    }
 }
```

Comment:
Missing newline

Reply:
Sorry, but where the extra newline is expected? (I checked the surrounding classes around, and the ones I checked neither have newline at the end of the file or after the last method)